sdk/formrecognizer/Azure.AI.FormRecognizer/src/ReadingOrder.cs

```diff
@@ -1,14 +1,30 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-namespace Azure.AI.FormRecognizer.Models
-{
-    // Setting the "internal" access modifier while we don't expose the reading order
-    // feature as part of our design.
+using Azure.Core;
 
-    internal enum ReadingOrder
+namespace Azure.AI.FormRecognizer
+{
+    /// <summary>
+    /// Specifies the order in which recognized text lines are returned in calls to
+    /// recognize content. This can be set in the <see cref="RecognizeContentOptions"/>.
+    /// As the sorting order depends on the detected text, it may change across images
+    /// and OCR version updates. Thus, business logic should be built upon the actual line
+    /// location instead of order.
+    /// </summary>
+    [CodeGenModel("ReadingOrder")]
+    public enum ReadingOrder
     {
+        /// <summary>
+        /// The lines are sorted top to bottom, left to right, although in certain cases
+        /// proximity is treated with higher priority.
+        /// </summary>
         Basic,
+
+        /// <summary>
+        /// The lines are returned in a more human-friendly reading order. Related text lines
+        /// are likely to be in sequence.
```

Comment:
I don't feel comfortable generalizing about `human-friendly reading order` without knowing if it actually includes everyone and if it adapts according to the language.
The service also explains it like `.  The new algorithm uses positional information to keep nearby lines together` so maybe taking something from there might be bette?

Reply:
Thank you for pointing this out. If we find out it doesn't include all FR supported languages, we may need to update it in the [documentation](https://docs.microsoft.com/en-us/azure/cognitive-services/form-recognizer/whats-new#march-2021) as well.

Reply:
for sure!

Reply:
The text I used was "uses heuristics to mimic the way a human reader would read the document." Maybe we should workshop this briefly to figure out the right way to communicate the difference between the basic and natural algorithms.

Reply:
Could either of you start a conversation in our Teams chat so we can get all languages in agreement on what the docstring should be?

Reply:
Updated to `The algorithm uses positional information to keep nearby lines together`.